source/app/util/environment-check.ts

```diff
@@ -132,7 +132,14 @@ export default async function environmentCheck (): Promise<void> {
     }
   }
 
-  process.env.ZETTLR_IS_TRAY_SUPPORTED = (await isTraySupported()).value.toString()
+  try {
+    if (await isTraySupported()) {
+      process.env.ZETTLR_IS_TRAY_SUPPORTED = '1'
```

Comment:
This looks much better. However, I'd still argue in here you should just put the following:

```js
process.env.ZETTLR_IS_TRAY_SUPPORTED = await isTraySupported()
```

Otherwise, when isTraySupported returns _false_, the environment variable will not even be set at all.

If you use the oneliner I provided above, however, the variable will be set, and only if the function throws an error do you have to manually set it to '0' and log the warning.

Reply:
Good idea. Your suggested code needs a slight change because can't assign `Boolean` to `String | undefined` 

``` typescript
    process.env.ZETTLR_IS_TRAY_SUPPORTED = await isTraySupported() ? '1' : '0'
```

Reply:
Absolutely fine!